laser_topic: "velodyne_points"
output_topic: "sync_data_out"
cam_1_id: "cam_1"
cam_2_id: "cam_2"
